tutorials/astropy-coordinates/2-Coordinates-Transforms.ipynb

```diff
@@ -716,6 +717,55 @@
    "source": [
     "In this tutorial, we have introduced the key concepts that underly the `astropy.coordinates` framework: coordinate component formats, representations, and frames. We demonstrated how to change the representation of a `SkyCoord` object (e.g., Cartesian to Cylindrical). We then introduce the concept of coordinate frames and frame transformations."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exercises"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Przybylski’s star or HD101065 is in the southern constellation of Centaurus with a right ascension of 174.4040348 degrees and a declination of -46.70953633 degrees. Create a Skycoord object of its sky position. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the distance to  of Przybylski’s star is 108.4 pc, retrieve a 3D Cartesian representation.(Hint: we did this earlier in the tutorial and it may help to create a new 3D Skycoord object.) "
```

Comment:
```suggestion
    "If the distance to Przybylski’s star is 108.4 pc, retrieve a 3D Cartesian representation. (Hint: we did this earlier in the tutorial and it may help to create a new 3D `SkyCoord` object.) "
```